README.md

```diff
@@ -129,7 +190,21 @@ SQL database. If you need an example, you can use `people` and `wishes` models.
 Your answer...
 
 ```
-Replace this with your answer
+Schema - is the organization or structure for a database.
```

Comment:
I didn't knew that. I thought it might also include the rows and the relationship between the columns and rows.
Got it, thanks.